ixmp/backend/jdbc.py

```diff
@@ -174,7 +179,9 @@ def __init__(self, jvmargs=None, **kwargs):
             dbprops = Path(kwargs.pop('dbprops'))
             if dbprops.exists() and dbprops.is_file():
                 # Existing properties file
-                properties_file, info = _read_dbprops(dbprops)
+                properties = _read_properties(dbprops)
+                if 'jdbc.url' not in properties:
```

Comment:
@khaeru can you check whether this if can work?

Reply:
Added 293d855 anyway.